source/Playnite/Common/Programs2.cs

```diff
@@ -211,21 +201,8 @@ public static async Task<List<Program>> GetShortcutProgramsFromFolder(string pat
                         continue;
                     }
 
-                    // Ignore uninstallers
-                    var targetFileName = Path.GetFileName(target);
-                    if (IsFileUninstaller(targetFileName))
-                    {
-                        continue;
-                    }
-
-                    // Ignore config and redistributables executables
-                    if (IsFileConfigOrRedistributable(targetFileName))
-                    {
-                        continue;
-                    }
-
-                    // Ignore crash handlers and game engine executables
-                    if (IsFileEngineExecutable(targetFileName))
+                    // Ignore uninstallers, config, redistributables and game engine executables
+                    if (IsFileUninstaller(Path.GetFileName(target)))

```

Comment:
IsFileUninstaller should be renamed to something more general since it's now not just about uninstaller files.

Reply:
huh. That's so weird, I already changed that in https://github.com/JosefNemec/Playnite/pull/2920/commits/f7753378c3dbe3cfdf37a432c241b9f7ec4145a0

I don't know why you still can see it but that name is no longer there

![image](https://user-images.githubusercontent.com/1389286/168188544-6e13c6b2-765c-41d3-819e-1f7cece04910.png)